functions/mqhandlers.go

```diff
@@ -62,7 +62,6 @@ func NodeUpdate(client mqtt.Client, msg mqtt.Message) {
 	logger.Log(0, "network:", newNode.Network, "received message to update node "+newNode.ID.String())
 	// check if interface needs to delta
 	ifaceDelta := wireguard.IfaceDelta(&node, &newNode)
-	keepaliveChange := node.PersistentKeepalive != newNode.PersistentKeepalive
 	//nodeCfg.Node = newNode
```

Comment:
why is this line being deleted

Reply:
It is only used below this in a removed "if" statement. "if keepaliveChange -> do stuff to netmaker.conf". VSCode told me it is unused after removing the if.